src/main/java/org/fundacionjala/coding/denis/Ean.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.denis;
+
+/**
+ * This is the main class of the European Article Number.
+ */
+public class Ean {
+    private String digits;
+    private static final int SIZECONTROL = 13;
+    private static final int EANNUMBER = 10;
+    private static final int MULTIPLICATOR = 3;
+    private static final int MODULO = 2;
+
+
+    /**
+     * eanVerify of the European article number.
+     *
+     * @return the eanVerify of the European article number.
+     */
+
+    public boolean eanKata() {
+        return getDigits().length() == SIZECONTROL && eanVerify();
+    }
+
+    /**
+     * executes the eanVerify of the European article number.
+     *
+     * @return executes the eanVerify of the European article number.
+     */
+    public boolean eanVerify() {
```

Comment:
Please join eanKata and eanVerify in a single method.
and remove getDigits and setDigits methods.